### PR TITLE
Product-cards task

### PR DESCRIPTION
**Pull request by mttony**
[DEMO LINK](https://mttony.github.io/layout_product-cards/)
[TEST REPORT LINK](https://mttony.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/stars.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+  
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+}
```
Comment (SanyaBratashchuk) [inline src/styles/stars.css:21]: image

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/styles/card.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
+.card {
+  max-width: 200px;
```
Comment (Bohdan-Maksymiuk) [inline src/styles/card.css:2]: why max-width?

File: src/styles/card.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  padding: 0 16px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-darkblue);
+}
+  
+.card__product-code {
+  margin: 4px 0 16px;
+  padding: 0 16px;
```
Comment (Bohdan-Maksymiuk) [inline src/styles/card.css:28]: could we set padding for the whole card instead of set for each element?

File: src/styles/card.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+  margin-top: 24px;
+  padding: 0 16px;
+}
+  
+.card__price-header {
+  margin: 0;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  padding-right: 55px;
```
Comment (Bohdan-Maksymiuk) [inline src/styles/card.css:63]: this custom padding looks not good - try to avoid such custom values

**Review by Bohdan-Maksymiuk — CHANGES_REQUESTED** [review]
Good job - try to fix comments

File: src/styles/card.css (hunk 2 above)
Comment (mttony) [reply src/styles/card.css:2]: should be just width?

File: src/styles/card.css (hunk 3 above)
Comment (mttony) [reply src/styles/card.css:28]: yeah, way better
fixing it now

File: src/styles/card.css (hunk 2 above)
Comment (mttony) [reply src/styles/card.css]: fixed

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,63 @@
...
+          class="card__buy-button" 
+          data-qa="hover" 
+          href="#"
+        >
+          buy
+        </a>
+      </div>
+    </div>
+  </body>
+
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:62]:
```suggestion
  </body>
```

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,63 @@
...
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+       class="card__product-picture" 
+       src="./images/imac.jpeg"
+       alt="apple_A1419_iMac27">
```
Comment (Bohdan-Maksymiuk) [inline src/index.html:18]:
```suggestion
      <img 
       class="card__product-picture" 
       src="./images/imac.jpeg"
       alt="apple_A1419_iMac27"
     >
```

**Review by Bohdan-Maksymiuk — APPROVED** [review]
pay attention to code style 😉

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**